### PR TITLE
adding group

### PR DESCRIPTION
**Pull request by kevvin2**

### CODE REVIEW

File: config/groups.yaml [hunk 1]
```diff
@@ -578,3 +578,9 @@
     - "2614670"
     - "2603372"
   id: 190031
+  - name: "SEPP Assignment Group"
+  members:
+    - "2629286"
```
Comment (mbg) [inline config/groups.yaml:583]: This isn't valid YAML, it would need to be:
```suggestion
- name: "SEPP Assignment Group"
  members:
    - "2629286"
```

**Review by mbg — CHANGES_REQUESTED** [review]

**Review by mbg — CHANGES_REQUESTED** [review]
It looks like `2629286` is already in a group `SEPP Assignment Prototype` (which I think was imported from the self sign-up groups). You have two options:

1. Instead of adding a new group, add the extra members to the existing group (the one with id `190011`)
1. Remove the existing group from `groups.yaml`

**Comment by mbg** [comment]
> But I don't have the option to push the changes? 

You should be able to push anything you want to your branch. In the [Files changed tab](https://github.com/uob-sepp/groups-2024/pull/85/files), which shows the cumulative changes of your PR, it does look like you have added the members to the existing group, but the new group is also still there.

**Comment by kevvin2** [comment]
Sorry I meant merge to the original repo from my fork by 'push'. I deleted the old group now, which did not have an id, and left the new group with an id with the added members from my last commit, is it now correct?

**Review by mbg — APPROVED** [review]

**Pull request merged**